src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1154,7 +1157,12 @@ class UpdateNodesBase : public IUpdateNodes {
             for (size_t i = node_indx; i < stop_indx; i++) {
                 const auto& node = m_executableGraphNodes[i];
                 if (node->isDynamicNode()) {
-                    node->updateShapes();
+                    try {
+                        node->updateShapes();
+                    } catch (const std::runtime_error& exp) {
+                        OPENVINO_THROW("Shape update of ", node->getName(),
+                            " node of type ", node->getTypeStr(), " failed ", exp.what());
+                    }
```

Comment:
Suggest wrap `updateNodes->run(stopIndx);` (L1334) in the try/catch construct as it the common entry point of shapeInfer/prepareParams subroutines.

Reply:
I agree, but anyway, we must provide node information on the level, where it is accessible, so I introduced 2 error handling levels at the graph stage and at node iteration

Reply:
Resolved after meeting